math/check_factorial.cpp

```diff
@@ -1,64 +1,72 @@
 /**
  * @file
  * @brief A simple program to check if the given number is a factorial of some
```

Comment:
```suggestion
 * @brief A simple program to check if the given number is a [factorial](https://en.wikipedia.org/wiki/Factorial) of some
```
hyperlinks seem more neat in this case